Buddies/Utilities/AlgoliaSearch.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  AlgoliaSearch.swift
+//  Buddies
+//
+//  Created by Noah Allen on 2/12/19.
+//  Copyright © 2019 Jack and the Beans. All rights reserved.
+//
+
+// Usage example:
+// let formatter = DateFormatter()
+// formatter.dateFormat = "yyyy/MM/dd"
```

Comment:
Now, roughly speaking, we can do "yyyy/MM/dd".toDate()

Reply:
Do I need to import anything to make that happen?

Reply:
Got it working 👍